### PR TITLE
[doc]intro to the compiling variables-EN

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
## What changes were proposed in this pull request?
Intro on the compiling variables. Translating from #1464

### CODE REVIEW

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 1]
```diff
@@ -76,8 +76,58 @@ Or you can build the release version.
...
 bash> cmake -DCMAKE_BUILD_TYPE=Release ..
-bash> make
-bash> sudo make install
+bash> make -j$(nproc)
+bash> sudo make install -j$(nproc)
+```
+
+#### Instructions on Variables
+
+- `CMAKE\_INSTALL\_PREFIX`: specifies the installation directory.
```
Comment (laura-ding) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: remove `` or \

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 2]
```diff
@@ -76,8 +76,58 @@ Or you can build the release version.
...
+
+For example, the following commands install **Nebula Graph** in the `/home/test` directory.
+
+```bash
+bash> cd nebula/build
+bash> cmake -DCMAKE_INSTALL_PREFIX=/home/test ..
+bash> make -j$(nproc) && make install -j$(nproc)
+```
+
+- `NEBULA\_BUILD\_VERSION`: set the version number for the bin files
```
Comment (laura-ding) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: ditto

modify `bin` to `binary`

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 3]
```diff
@@ -76,8 +76,58 @@ Or you can build the release version.
...
+bash> make -j$(nproc) && make install -j$(nproc)
+```
+
+The following command checks the version number of the compiled bin files in `/usr/local/nebula/bin/` directory.
+
+```bash
+bash> /usr/local/nebula/bin/nebula --version
+```
+
+- `ENABLE\_TESTING`: specifies whether to compile the UT code.
```
Comment (laura-ding) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: ditto

File: docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md [hunk 4]
```diff
@@ -76,8 +76,58 @@ Or you can build the release version.
...
+
+The default value is ON. If you don't want to compile, set it to OFF during cmake with the following command.
+
+```bash
+bash> cd nebula/build
+bash> cmake -DENABLE_TESTING=OFF ..
+bash> make -j$(nproc) && make install -j$(nproc)
+```
+
+- `ENABLE\_ASAN`: Add AddressSanitizer when compiling.
```
Comment (laura-ding) [inline docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md]: ditto

**Comment by jude-zhu** [comment]
Jenkins go

**Pull request merged**